src/classes/GE_GiftEntryController.cls

```diff
@@ -964,7 +959,7 @@ public with sharing class GE_GiftEntryController {
             Custom_Column_Header__c columnHeader = new Custom_Column_Header__c();
             columnHeader.Field_Api_Name__c = field;
             columnHeader.Index__c = index;
-            columnHeader.Name = generateId();
+            columnHeader.Name = UTIL_String.getGUID();
```

Comment:
Using the new `UTIL_String.getGUID()` and removed the same code implemented in `generateId()`.

Reply:
I think this changer makes sense? (CC @psadie-sfdo )

Reply:
This looks good to me!